README.md

```diff
@@ -1,6 +1,6 @@
 # Python SDK for SecureDrop
 
-[![CircleCI](https://circleci.com/gh/freedomofpress/securedrop-sdk/tree/master.svg?style=svg)](https://circleci.com/gh/freedomofpress/securedrop-sdk/tree/master)
+[![CircleCI](https://circleci.com/gh/freedomofpress/securedrop-sdk/tree/master.svg?style=svg)](https://circleci.com/gh/freedomofpress/securedrop-sdk/tree/main)
```

Comment:
The URL to the svg file needs to be changed here as well: the link works, but the PNG is red/failed because it refers to the state of the now defuct `master` branch, see https://github.com/freedomofpress/securedrop-sdk/tree/update-readme

Reply:
a-ha! at the time i left this because the master branch in ci was correct, but now that you fixed up ci you are correct. good eye good eye... updating